src/interpolation.jl

```diff
@@ -85,10 +85,33 @@ function interpolate_kpoint(data_in::AbstractVecOrMat, kpoint_in::Kpoint, kpoint
     data_out
 end
 
+"""
+Compute indices from basis_in that are in basis_out, alongside the complementary
+set of indices of basis_out that are not in basis_in.
+"""
```

Comment:
makes it sound like it returns two things but it just returns one?

Reply:
Also extremely unclear what this function is supposed to do. Is that two bases for the same system? (in which case add a check for that). Are you talking about the k-dependent bases or the global one? The indices are in what array?

Reply:
> makes it sound like it returns two things but it just returns one?

woups yes, my mistake

> Also extremely unclear what this function is supposed to do. Is that two bases for the same system? (in which case add a check for that). Are you talking about the k-dependent bases or the global one? The indices are in what array?

I'll try to make it clearer.